src/kOS/Module/AssetManager.cs

```diff
@@ -144,6 +139,22 @@ public Font GetSystemFontByNameAndSize(string name, int size, bool checkMono, bo
                     ScreenMessages.PostScreenMessage(
                         string.Format("<color=#ff9900><size=20>{0}</size></color>", msg), 8, ScreenMessageStyle.UPPER_CENTER);
                 }
+                // Must destroy the font right now, else Unity keeps the font data temporarily alive for
+                // too long a window of time, and it ends up failing when there's too many of
+                // them loaded at once.  Relying on the cleanup that automatically happens when you
+                // orphan the Font takes too long to avoid this bug.  As long as it *temporarily* had
+                // too much font data in memory, it breaks some of the other fonts that were loaded but
+                // hadn't be excercised yet.  (i.e nothing drawn in Arial with default GUI.skin can be
+                // seen anymore).
+                // This is a bug that during our user community testing didn't seem to happen to everybody,
+                // but it happened to some users depending on kinds of fonts they had on their OS and kinds
+                // of graphics cards (it seems to be related to the texture size of the texture being sent to
+                // the graphics card).  At any rate, explicitly doing this instead of waiting for the
+                // cleanup that is epxected to "magically" happen later on its own seems to cure the problem:
```

Comment:
So, calling `Destroy` won't actually clean up the underlying unmanaged resources?  That's not all that useful.  I hope that the resources aren't reallocated if we create another instance of a wrapper around the same thing, otherwise that's pretty much a memory leak by design.